### PR TITLE
Issue #7613: Update doc for MutableException 

### PR DESCRIPTION
**Pull request by ShubhraSuman**
Issue: #7613 add code example:

image

### CODE REVIEW

**Comment by nick-mancuso** [comment]
@ShubhraSuman please follow all rules in PR description template.

**Comment by ShubhraSuman** [comment]
> @ShubhraSuman please follow all rules in PR description template, and instructions at https://checkstyle.org/contributing.html

**Comment by nick-mancuso** [comment]
@ShubhraSuman please fix PR title and make CI happy.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java [hunk 1]
```diff
@@ -66,6 +66,34 @@
...
+ *      public FirstClassNotExtendError() {
+ *           errorCode = 1;
+ *       }
+ *   }
+ *
+ *   class CustomException extends java.lang.Exception {} // OK no member variable
+ *
+ *   class SecondMutableException extends java.lang.Exception {
+ *       int errorCode; // violation needs to be final
+ *   }
```
Comment (baratali) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java:95]: please fix code indentation. it's not consistent

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java [hunk 2]
```diff
@@ -66,6 +66,34 @@
  * <pre>
  * &lt;module name=&quot;MutableException&quot;/&gt;
```
Comment (baratali) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java:67]: Please add two more configs and examples for each property with a custom value: `format` and `extendedClassNameFormat`

**Review by baratali — CHANGES_REQUESTED** [review]
Please rename your commit message to follow correct format:
```
[ERROR]   CommitValidationTest.testCommitMessageHasProperStructure:189 Commit b89673c5030196128689d632582f801065f0f5e5 message: "Fix: Changes for MutableException-7613\n" is invalid
Proper commit message should adhere to the following rules:
    1) Must match one of the following patterns:
        ^Issue #\d+: .*$
        ^Pull #\d+: .*$
        ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    2) It contains only one line of text
    3) Must not end with a period, space, or tab
    4) Commit message should be less than or equal to 200 characters
```

**Comment by ShubhraSuman** [comment]
> Hi @ShubhraSuman. Do you need any help with this PR? Please let us know if you're going to continue this pull request. Thanks.

Hi, I am continuing this PR.

**Comment by baratali** [comment]
Commit message is `Fix: Changes for MutableException-7613\n`, but it should be `Issue #7613: Update doc for MutableException`

**Comment by ShubhraSuman** [comment]
> Commit message is `Fix: Changes for MutableException-7613\n`, but it should be `Issue #7613: Update doc for MutableException`
Its already this.
image

**Comment by baratali** [comment]
image

**Comment by ShubhraSuman** [comment]
> image
Done. Thanks

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java (hunk 1 above)
Comment (ShubhraSuman) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java]: Done

Comment (baratali) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java]: there are still problems with indentation.
example:
image

**Comment by baratali** [comment]
Please squash your commits into one. There are currently 3 different commits.
Please read: https://github.com/checkstyle/checkstyle/wiki/PR-rules

**Comment by nick-mancuso** [comment]
We have lost contact with author of PR, anyone is welcome to re-use changes here to help solve this issue.

**Pull request closed without merging**